pyprland/plugins/expose.py

```diff
@@ -23,7 +23,10 @@ async def run_expose(self) -> None:
         If expose is active restores everything and move to the focused window
         """
         if self.exposed:
-            commands = [f"movetoworkspacesilent {client['workspace']['id']},address:{client['address']}" for client in self.exposed_clients]
+            commands = [
```

Comment:
you may want to run "ruff format" on the files, it's part of the pre-commit hooks in theory...

Reply:
Apologies, it should be all good now  :+1: .